less/transferwise.less

```diff
@@ -20,6 +20,25 @@
 @brand-warning-hover: #DF8701;
 //@brand-danger-hover:
 
+
+h1 {
+  font-size: 36px;
+  line-height: 48px;
+  font-weight: 600;
+}
+
+h3 {
+  line-height: 34px;
+  font-weight: 400;
+  margin-top: 10px;
+}
+
+p {
+  font-size: 14px;
+  line-height: 22px;
```

Comment:
Actually, thinking about this a bit more, I might handle this one.  Line-height is a sensitive change as things like buttons should work within the context of a line of text, so these need to be considered at the same time.